infra/testApi.mjs

```diff
@@ -17,17 +17,31 @@ assert.equal(improveResponse.status, 200)
 // assert.equal(improveHTML.length, improveHTMLexpectedCount, `HTML response character count should be ${improveHTMLexpectedCount} unless HTML changes.`)
 
 // improve-start endpoint
-const startURL = `http://127.0.0.1:8000/improve-start?formula=${encodeURIComponent(FPCoreFormula2)}`
-const startResponse = await fetch(startURL, { method: 'POST' })
-assert.equal(startResponse.status, 201)
+const startURL = `http://127.0.0.1:8000/improve-start`
+const URIencodedBody = "formula=" + encodeURIComponent(FPCoreFormula)
+const startResponse = await fetch(startURL, {
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/x-www-form-urlencoded',
+  },
+  body: URIencodedBody
+})
+const testResult = startResponse.status == (201 || 202)
```

Comment:
No, this doesn't work at all! This first computes `201 || 202`, which (because `201` is truthy, returns `201`) and then returns that.

Reply:
TIL what truthy is. 😭